homeassistant/components/lamarzocco/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""The La Marzocco integration."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.helpers import config_validation as cv
+
+from .const import DOMAIN
+from .coordinator import LmApiCoordinator
+
+PLATFORMS = [
+    Platform.WATER_HEATER,
+]
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up La Marzocco as config entry."""
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator = LmApiCoordinator(
```

Comment:
```suggestion
    coordinator = LmApiCoordinator(
```

Please add the coordinator to hass.data after calling `await coordinator.async_config_entry_first_refresh()`